dragonchain/lib/broadcast.py

```diff
@@ -36,6 +37,8 @@
 
 def dispatch(block: "model.BlockModel") -> None:
     if LEVEL == "5":
+        _log.info("Trying to finalize any previously failed claims...")
+        process_claims_backlog()
```

Comment:
You can't just make an async call. All this does is return a coroutine. This will do literally nothing.

I think you need to look up how to use asyncio and python...

Reply:
Don't know why I originally set this up with some async stuff, I don't see a particular need for it.  Wasn't ready for PR review, fixed.